### PR TITLE
New example and tutorial showing how to create a live scrolling plot

### PR DESCRIPTION
**Pull request by jleibs**
### What
- Resolves: https://github.com/rerun-io/rerun/issues/5820

Although I would like to update the blueprint_stocks example for completeness, we decided it would be nice to have a tight live-scrolling example and how-to guide since this is one of the more common things people look for.

Guide:
 - https://landing-e9j32r1lf-rerun.vercel.app/docs/howto/fixed-window-plot

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6314?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6314?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6314)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: docs/content/howto/fixed-window-plot.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Create a fixed-window plot
+order: 150
+---
+
+As of Rerun 0.16, the [TimeSeriesView](../reference/types/views/time_series_view.md) now supports direct
+manipulation of visible time range. This allows you to create a plot that only shows a fixed window of data.
```
Comment (teh-cmc) [inline docs/content/howto/fixed-window-plot.md:7]:
```suggestion
manipulation of the visible time range. This allows you to create a plot that only shows a fixed window of data.

File: examples/python/live_scrolling_plot/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Install the necessary libraries specified in the requirements file:
+```bash
+pip install -e examples/python/live_scrolling_plot
+```
+
+Then, simply execute the main Python script:
+```bash
+python -m live_scrolling_plot
+```
+ddd
```
Comment (teh-cmc) [inline examples/python/live_scrolling_plot/README.md:70]: ddd

**Review by teh-cmc — APPROVED** [review]
very nice

File: examples/python/live_scrolling_plot/README.md (hunk 2 above)
Comment (emilk) [reply examples/python/live_scrolling_plot/README.md:70]: I sense vim

Comment (jleibs) [inline examples/python/live_scrolling_plot/README.md:70]:
```suggestion
```

**Pull request merged**